README.md

```diff
@@ -105,6 +109,7 @@ The validator throws errors at you! Why? Assuming you haven't made any mistakes
 
 ```
 Your answer...
+the erb tags are incomplete. Instead of "<%" there needs to be an "<%=" for the beginning half of the erb tags.
```

Comment:
Please review.  What does the HTML validator validate?  Is this the right format for that?  When do we have the right format?